### PR TITLE
Add insights sorting to host index page

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by laviro** [comment]
Ready for review @ShimShtein

**Review by ShimShtein — CHANGES_REQUESTED** [review]
Two additional issues:

- [ ] Hosts that didn't have insights facet objects do not cache `inisghts_hits` correctly

- [ ] When we sort `desc`, `nil` values appear on top, instead of the bottom.

**Comment by ShimShtein** [comment]
@adamruzicka, if I read correctly `scoped_search` [builder](https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/query_builder.rb#L587) of order by statements, there is no way we can influence `null first`/`null last` definition?

**Comment by adamruzicka** [comment]
@ShimShtein I read it the same way :/

File: app/overrides/hosts_list.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   virtual_path: 'hosts/_list',
   name: 'insights_hits_header',
   insert_before: 'thead tr th.hidden-xs:first-of-type',
-  text: '<th class="hidden-xs" width="10%"><%= _("Insights")%></th>'
+  text: '<th class="hidden-xs" width="10%"><%= sort :"insights_recommendations_count", :as => _("Insights")%></th>'
```
Comment (ShimShtein) [inline app/overrides/hosts_list.rb:5]:
```suggestion
  text: '<th class="hidden-xs" width="10%"><%= sort :insights_recommendations_count, :as => _("Insights")%></th>'
```

No need for string here.

**Review by ShimShtein — CHANGES_REQUESTED** [review]
Small nitpick and we're done.

**Comment by ShimShtein** [comment]
Merged, thanks @laviro!

**Pull request merged**